safe_transaction_service/history/signals.py

```diff
@@ -247,9 +247,13 @@ def process_webhook(
     )
     for payload in payloads:
         if address := payload.get("address"):
-            send_webhook_task.delay(address, payload)
+            send_webhook_task.apply_async(
+                args=(address, payload), priority=1
+            )  # Almost the lowest priority
             if is_relevant_notification(sender, instance, created):
-                send_notification_task.apply_async(args=(address, payload), countdown=5)
+                send_notification_task.apply_async(
+                    args=(address, payload), countdown=5, priority=1
```

Comment:
Isn't the highest priority `0`? If we only use `0` so far then I think `1` makes sense (or If it needs to be a higher number ie.: lower priority)

Reply:
It looks like for Redis is the opposite 🤦🏻 . Thanks for noticing @fmrsabino, I have changed it